### ISSUE TITLE
sihas climate error message after updating to v1.6.2b1

### ISSUE THREAD

**Issue by sungokay** [cmsong-shina/sihas-canary#14]
Hello, 

I had updated a custom component of sihas to v1.6.2b1 and met a below error message.
But, After rollbacked to V1.6.1. the problem had disappeared.
HA's version is 2024.1.3.

Thanks
SungOk

```
2024-01-14 15:31:12.550 ERROR (MainThread) [homeassistant.helpers.entity] Update for climate.home_heater fails
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 898, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1218, in async_device_update
    await hass.async_add_executor_job(self.update)
  File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/sihas/climate.py", line 457, in update
    self._attr_hvac_action = self._resolve_hvac_action(regs)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/sihas/climate.py", line 490, in _resolve_hvac_action
    return HVACAction.HEAT
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/enum.py", line 784, in __getattr__
    raise AttributeError(name) from None
AttributeError: HEAT
```
'

**Comment by cmsong-shina** [comment]
Can you try with v1.6.2b2?

**Comment by sungokay** [comment]
Thanks for your update but I had met another problem after updated to v1.6.2b2 as a below.

SungOk.


```
2024-01-15 13:29:13.908 WARNING (MainThread) [homeassistant.const] TEMP_CELSIUS was used from sihas, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.CELSIUS instead, please create a bug report at https://github.com/cmsong-shina/sihas-canary/issues
2024-01-15 13:29:13.920 WARNING (MainThread) [homeassistant.const] ENERGY_KILO_WATT_HOUR was used from sihas, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfEnergy.KILO_WATT_HOUR instead, please create a bug report at https://github.com/cmsong-shina/sihas-canary/issues
2024-01-15 13:29:13.923 WARNING (MainThread) [homeassistant.const] POWER_WATT was used from sihas, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfPower.WATT instead, please create a bug report at https://github.com/cmsong-shina/sihas-canary/issues
2024-01-15 13:29:13.926 WARNING (MainThread) [homeassistant.const] TEMP_CELSIUS was used from sihas, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.CELSIUS instead, please create a bug report at https://github.com/cmsong-shina/sihas-canary/issues
2024-01-15 13:29:13.929 WARNING (MainThread) [homeassistant.const] ELECTRIC_POTENTIAL_VOLT was used from sihas, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfElectricPotential.VOLT instead, please create a bug report at https://github.com/cmsong-shina/sihas-canary/issues
2024-01-15 13:29:13.933 WARNING (MainThread) [homeassistant.const] ELECTRIC_CURRENT_AMPERE was used from sihas, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfElectricCurrent.AMPERE instead, please create a bug report at https://github.com/cmsong-shina/sihas-canary/issues
2024-01-15 13:29:13.936 WARNING (MainThread) [homeassistant.const] FREQUENCY_HERTZ was used from sihas, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfFrequency.HERTZ instead, please create a bug report at https://github.com/cmsong-shina/sihas-canary/issues
```

**Comment by cmsong-shina** [comment]
I believe all deprecated constant waring is solved on  v1.6.2b3.

[Deprecated constants create a log entry | Home Assistant Developer Docs](https://developers.home-assistant.io/blog/2023/12/19/constant-deprecation)

**Comment by sungokay** [comment]
The problem had been disappeared at v1.6.2.b3.
Thanks for your deep support.

SungOk
